types/react-window/index.d.ts

```diff
@@ -264,6 +264,10 @@ export interface GridProps extends CommonProps {
      * Called when the grid scroll positions changes, as a result of user scrolling or scroll-to method calls.
      */
     onScroll?: (props: GridOnScrollProps) => any;
+    /**
+     * @deprecated since version 1.8.2, please use the overscanColumnCount
```

Comment:
Nitpick: could remove the word “the”

Reply:
Yes, sorry, that was a copy-paste from the code... https://github.com/bvaughn/react-window/pull/229/files#diff-3845a69a9768c7a65c701ae9243d1bc9R276